### PR TITLE
Fix alpha flag access

### PR DESCRIPTION
**Pull request by 8enSmith**
Refactored the code to use the flags.get() function to access a flag value.

### CODE REVIEW

**Review by taraojo — APPROVED** [review]
Thanks Ben 🎉

**Review by fenglish — APPROVED** [review]
Thank you!

File: test/navigationAlphaTest.spec.js [hunk 1]
```diff
@@ -31,12 +31,14 @@ describe('navigationAlphaTest', () => {
...
 				flags:
 				{
+					// eslint-disable-next-line no-unused-vars
 					get: (_flag) => { return 'on'; }
 				}
 			} :
 			{
 				flags :
 				{
+					// eslint-disable-next-line no-unused-vars
```
Comment (taraojo) [inline test/navigationAlphaTest.spec.js:41]: Is it worth removing the unused `_flag` instead of disabling the linter?

Comment (8enSmith) [reply test/navigationAlphaTest.spec.js:41]: The get function takes an input parameter but I'm not using it. I'll look to see if I can work around it.

Comment (taraojo) [reply test/navigationAlphaTest.spec.js]: I guess it gives clarity to how the function works, so we can leave this as is and merge 🙂

Comment (8enSmith) [reply test/navigationAlphaTest.spec.js]: I've removed the redundant flag

**Pull request merged**